deps/rabbit/test/rabbit_fifo_SUITE.erl

```diff
@@ -1751,8 +1750,11 @@ single_active_consumer_priority_test(Config) ->
                when map_size(Ch) == 0)
 
     ],
-    {_S1, _} = run_log(Config, S0, Entries, fun single_active_invariant/1),
-
+    {#rabbit_fifo{ cfg = #cfg{resource = Resource}}, ModCalls} = run_log(Config, S0, Entries, fun single_active_invariant/1),
+    {mod_call,rabbit_quorum_queue,update_consumer_handler,
+            [Resource,
+            _,
+            false,undefined,undefined,false,waiting,[]]} = lists:nth(3, ModCalls),
```

Comment:
You added an assertion for only the 3rd effect.
Can we add an assertion that checks for **all** expected `mod_call` effects updating the consumers' activity statuses? Even the order of the returned effects can be asserted.
This way, test coverage for updating the consumer activity status will be complete and the test will be easier to understand.